### PR TITLE
add datasources, refactor example blockstorage

### PR DESCRIPTION
**Pull request by volosatik**
closes CMPT-33762
closes CMPT-33763

### CODE REVIEW

File: examples/blockstorage/snapshot/main-datasource.tf [hunk 1]
```diff
@@ -0,0 +1,5 @@
+data "vkcs_blockstorage_snapshot" "recent_snapshot" {
+  name        = "snapshot-tf-example"
+  most_recent = true
+  depends_on  = [vkcs_blockstorage_snapshot.recent_snapshot]
```
Comment (ftersin) [inline examples/blockstorage/snapshot/main-datasource.tf:4]: add the comment here about this dependency

**Comment by ftersin** [comment]
move import.sh out of examples folder

File: examples/blockstorage/snapshot/main-datasource.tf (hunk 1 above)
Comment (volosatik) [reply examples/blockstorage/snapshot/main-datasource.tf:6]: done, and in volume

**Comment by volosatik** [comment]
> move import.sh out of examples folder

move to templates/(named folder)

**Review by ftersin — APPROVED** [review]

**Pull request merged**